File: adk-python/agent.py
import argparse
from src.interfaces.cli_runner import BenchmarkRunner

if __name__ == "__main__":
    parser = argparse.ArgumentParser(description="Uruchomienie Benchmarku Invoice Agenta")
    parser.add_argument("--debug", action="store_true", help="Włącza podgląd na żywo z myślenia agenta i narzędzi")
    args = parser.parse_args()

    BenchmarkRunner.execute_all(debug=args.debug)

File: adk-go/agent.go
package main

import (
	"flag"
	"fmt"

	"adk-benchmark/src/interfaces"
)

func main() {
	debugFlag := flag.Bool("debug", false, "Włącza podgląd na żywo z myślenia agenta i narzędzi")
	flag.Parse()

	if *debugFlag {
		fmt.Println("\033[1;36m[TRYB DEBUG AKTYWNY]\033[0m")
	}

	interfaces.ExecuteAll(*debugFlag)
}